tracer/test/test-applications/integrations/Samples.DataStreams.Kafka/Samples.DataStreams.Kafka.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Confluent.Kafka" Version="1.4.3" />
```

Comment:
I've updated the version since there's a bug in the older version of this library, causing `Produce` to fail on ARM OSX.

Reply:
It's probably best to revert this before the final push, so that we still test the oldest supported version by default 🙂